layouts/page/guideline.html

```diff
@@ -0,0 +1,204 @@
+{{- define "main" -}}
+    {{ partial "nav.html" (dict "nav_white" 1) }}
+    {{ $guideline := .Site.Data.guideline }}
+
+    <section style="padding-top: 6.5rem" class="mb-5">
+        <div class="container">
+            <div class="d-flex row justify-content-between align-items-center">
+                <!-- Left Image -->
+                <div class="col-md-2 text-center d-none d-md-block ms-1 me-0 mb-0 px-0 py-0" style="width: 15%">
+                    <img style="max-width: 100%;" src='{{ absURL "guideline/side-img-1.png" }}'/>
+                </div>
+                <!-- Center Content -->
+                <div class="col-md-8">
+                    <h2 class="text-center text-primary fw-lighter">
+                        Know each <span class="fw-bold">department</span>.
+                    </h2>
+                    <p class="text-center">
+                        Find out more about the mission and general tasks of
+                        each department.
+                    </p>
+                </div>
+                <!-- Right Image -->
+                <div class="col-md-2 text-center d-none d-md-block me-1 px-3" style="width: 15%">
+                    <img style="max-width: 100%;" src='{{ absURL "guideline/side-img-2.png" }}' />
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!--list of departments-->
+    <section class="bg-body-secondary mb-5">
+        <div class="container mt-2 py-5">
+            <!-- Executive Director row -->
+            <div class="row justify-content-center">
+                {{ $depFirstRow := slice "Executive Director" "Secretarial Director" }}
+                {{ range $depFirstRow }}
+                    <div class="col-12 col-md-4 mb-3">
+                        <div style="border-radius: 15px;" class="col bg-secondary py-3 px-3 d-flex align-items-center justify-content-center">
+                            <p class="text-primary mb-0" style="font-weight: 600;">
```

Comment:
```suggestion
                            <p class="text-primary mb-0 fw-semibold">
```
600 is `fw-semibold`

Reply:
bs doesn't include a fw-semibold class by default., and it didn't work when I tried it as well, perhaps we need to extend bs utility classes for this?

Reply:
> bs doesn't include a fw-semibold class by default., and it didn't work when I tried it as well, perhaps we need to extend bs utility classes for this?

@stingynoodles

afk

U mean the class still does not apply after rerun server? Is it the same prob as above?

If it is not, may u check in inspector tool, see if the class is applied to the html?

Reply:
@stingynoodles 

If so, u can safely delete line 33-46, 50, 58

Reply:
i tried again with npm run serve and it worked, might be some lagging or smtg, thankss